securing_apps/topics/oidc/java/client-authentication.adoc

```diff
@@ -53,6 +54,10 @@ For set up on the adapter side you need to have something like this in your `key
 With this configuration, the keystore file `keystore-client.jks` must be available on classpath in your WAR. If you do not use the prefix `classpath:`
 you can point to any file on the file system where the client application is running.
 
+The `algorithm` field specifies the algorithm used for the Signed JWT and it defaults to `RS256`. This field should be in sync with the key pair.
+For example algorithm RS256 needs RSA keys but ES256 requires EC keys. Please refer to
```

Comment:
How about the following?

For example, the `RS256` algorithm needs a RSA key pair while the `ES256` algorithm requires an EC key pair

Reply:
It's OK to me! Modified!